high_school/views.py

```diff
@@ -3,61 +3,115 @@
 from sodapy import Socrata
 
 from OneApply.constants import ApiInfo, UserType
+from high_school.models import Program
 from .forms import SaveHighSchoolsForm
 from .serializer import HighSchoolSerializer
 from .models import HighSchool
 
 client = Socrata(ApiInfo.API_DOMAIN, ApiInfo.APP_TOKEN)
 
 
-def save_highschool_data(request, user_type):
+def save_highschool_data(request):
```

Comment:
will there be a change in url since you are removing the parameter?

Reply:
The URL was already changed but the user_type was never removed.